src/System.Linq.Queryable/src/System.Linq.Queryable.csproj

```diff
@@ -32,5 +32,12 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\System.Linq\src\System.Linq.csproj">
```

Comment:
I'd thought that, but it was already in there. (I'm sure I'm doing something dim here).